src/_nebari/stages/kubernetes_services/template/modules/kubernetes/services/jupyterhub/files/jupyterhub/02-spawner.py

```diff
@@ -24,24 +24,44 @@ def get_username_hook(spawner):
 
 
 def get_conda_store_environments(user_info: dict):
+    import math
+
     import urllib3
     import yarl
 
     external_url = z2jh.get_config("custom.conda-store-service-name")
     token = z2jh.get_config("custom.conda-store-jhub-apps-token")
     endpoint = "conda-store/api/v1/environment"
 
-    url = yarl.URL(f"http://{external_url}/{endpoint}/")
-
+    size_limit = 100
+    url = yarl.URL(f"http://{external_url}/{endpoint}/?size={size_limit}")
```

Comment:
What are the implications of different size limits? What is the default for conda-store? Should this be a parameter that can be configured? If not, it should likely be a constant.

Reply:
> What are the implications of different size limits?

The size limit determines how many results are returned by the api. A bigger size limit will mean there is a bigger size response, but will need to make a smaller number of requests to get all of the envs. 

> What is the default for conda-store? 

The default for conda-store is 100.

> Should this be a parameter that can be configured? 

No, this is an implementation detail so should not be configurable by users. Can make it a constant :+1:

Reply:
Can you just leave a comment inline with the constant saying that it is the conda-store default so that we know why that number was put in?